witness/src/Dscp/Witness/Logic/Processing.hs

```diff
@@ -57,17 +58,20 @@ createBlock newSlot = do
 applyBlockRaw :: (WitnessWorkMode ctx m, WithinWriteSDLock) => Bool -> Block -> m AvlProof
 applyBlockRaw toVerify block = do
     (sdActions :: SDActions) <- view (lensOf @SDActions)
+    (sdOSParamsBuilder :: SD.OSParamsBuilder) <- view (lensOf @SD.OSParamsBuilder)
     let blockDBM = nsBlockDBActions sdActions
-    let stateDBM = nsStateDBActions sdActions (RememberForProof True)
+    let stateDBM = nsStateDBActions sdActions (SD.RememberForProof True)
 
     (blockCS, stateCS) <- reify blockPrefixes $ \(_ :: Proxy ps) ->
         let actions = SD.constructCompositeActions @ps (SD.dmaAccessActions blockDBM)
                                                         (SD.dmaAccessActions stateDBM)
             rwComp = do
               sblock <- SD.liftERoComp $ expandBlock block
-              SD.applyBlockImpl toVerify blkStateConfig sblock
+              -- getCurrentTime requires MonadIO
+              let osParams = SD.unOSParamsBuilder sdOSParamsBuilder $ UTCTime (toEnum 0) (toEnum 0)
```

Comment:
What does it mean/do?

Reply:
Now it does nothing. This feature is called OS paras handling and is needed e.g. to verify that all blocks in fork are not from the future slot. Unfortunately the way the feature was implemented doesn't allow for easy integration with DSCP, so we left this placeholder for now (and another one in `bcValidateFork`.